thorlcr/activities/null/thnullslave.cpp

```diff
@@ -75,7 +75,7 @@ class CNullSlaveActivity : public CSlaveActivity, public CThorDataLink
 
     virtual bool isGrouped()
     {
-        return queryHelper()->queryOutputMeta()->isGrouped();
+        return false;
```

Comment:
I'm not sure it matters.. because I can't think of anywhere where it would make any different the pulling activity, given there are 0 records.
But I'm also not sure I really had to make that change either